### PR TITLE
Getting started example go

### PR DESCRIPTION
**Pull request by jmbrown412**

### CODE REVIEW

**Comment by jmbrown412** [comment]
codecov/project  is failing probably because there is more go code in the project now and not accompanying tests for it?

File: README.md [hunk 1]
```diff
@@ -26,10 +26,15 @@ Start PayGate and its dependencies:
...
 
-Create an example Transfer
+Create an example Transfer with shell script
 ```
 ./examples/getting-started.sh
 ```
+or using Go
+```
+Linux/OSX: go run examples/getting-started/main.go
+Windows: go run examples\getting-started\main.go
```
Comment (adamdecaf) [inline README.md:36]: Can we use the same path for all three OS's?

File: README.md [hunk 2]
```diff
@@ -26,10 +26,15 @@ Start PayGate and its dependencies:
 docker-compose up
 ```
 
-Create an example Transfer
+Create an example Transfer with shell script
 ```
 ./examples/getting-started.sh
```
Comment (adamdecaf) [inline README.md:31]: I was hoping we delete this bash script.

File: examples/getting-started/main.go [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+	req, err := http.NewRequest("PUT", url, &body)
+	if err != nil {
+		return false, err
+	}
+	resp, err := httpClient.Do(req)
+	if err != nil {
+		return false, err
+	}
+	if resp.StatusCode == 200 {
+		return true, err
```
Comment (adamdecaf) [inline examples/getting-started/main.go:296]: This looks the same as
```go
return resp.StatusCode == 200
```

**Comment by adamdecaf** [comment]
Likely yea. It would be nice to run `docker-compose up` and then this script in CI.

File: README.md (hunk 2 above)
Comment (jmbrown412) [reply README.md:31]: I can do that. Wasn't sure if we wanted to keep it around.

File: examples/getting-started/main.go (hunk 3 above)
Comment (jmbrown412) [reply examples/getting-started/main.go:296]: I can change it.

File: README.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ docker-compose up
 
 Create an example Transfer
 ```
-./examples/getting-started.sh
+go run examples\getting-started\main.go
```
Comment (adamdecaf) [inline README.md:31]: I think `\` slashes only work on windows. Can we use `/` - I'm pretty sure Go supports that on Windows

```
$ go run .\cmd\server 
go: finding module for package .cmdserver
package .cmdserver: malformed module path ".cmdserver": leading dot in path element
```

Comment (jmbrown412) [reply README.md]: Done

**Comment by adamdecaf** [comment]
Thanks @jmbrown412 !

**Pull request merged**